docs/servicenow_setup.md

```diff
@@ -19,7 +19,25 @@ We need to set up the virtual agent on a ServiceNow instance, to which we will s
     https://<your-mattermost-url>/plugins/mattermost-plugin-servicenow-virtual-agent/api/v1/nowbot/processResponse?secret=<your-webhook-secret>
     ```
     **Note**: (Webhook secret can be generated from the Mattermost system console settings of the Virtual agent plugin.)
- 
+  - Adding "User-Agent" header:
```

Comment:
```suggestion
  - Adding "User-Agent" header:
  
```

Reply:
You can see this link and add the reason for adding this - 
https://support.servicenow.com/kb?id=kb_article_view&sysparm_article=KB0720934
https://www.servicenow.com/community/it-service-management-forum/403-forbidden-username-password-combo/m-p/770873/page/2